### PR TITLE
New web design and clean up documentation

### PR DESCRIPTION
**Pull request by ry**
I've already uploaded it to the main website https://deno.land/

### CODE REVIEW

File: website/Manual.md [hunk 1]
```diff
@@ -8,7 +10,71 @@ notice.
...
+- Uses "ES Modules" and does not support `require()`.
+
+- File system and network access can be controlled in order to run sandboxed
+  code. Access between V8 (unprivileged) and Rust (privileged) is only done via
+  serialized messages defined in this
+  [flatbuffer](https://github.com/denoland/deno/blob/master/src/msg.fbs). This
+  makes it easy to audit. For example, to enable write access use the flag
+  `--allow-write` or for network access `--allow-net`.
+
+- Only ship a single single executable.
```
Comment (jakeg) [inline website/Manual.md]: Word "single" repeated twice

File: website/Manual.md [hunk 2]
```diff
@@ -226,6 +300,30 @@ It's worth noting that like the `cat.ts` example, the `copy()` function here
...
+### File server
+
+This one serves a local directory in HTTP.
+
+```
+alias file_server="deno  --allow-net --allow-read \
+  https://deno.land/x/http/file_server.ts"
+```
+
+Then it `file_server`:
```
Comment (jakeg) [inline website/Manual.md]: ?

File: website/Manual.md (hunk 1 above)
Comment (ry) [reply website/Manual.md]: Fixed - thanks.

File: website/Manual.md (hunk 2 above)
Comment (ry) [reply website/Manual.md]: Fixed

File: website/style.css [hunk 3]
```diff
@@ -1,74 +1,86 @@
...
+svg {
+  margin: 0px;
+  width: 100%;
+  height: 300px;
 }
-a.badge:hover {
-  box-shadow: 0 0 5px 0 #333;
+
+a {
+  color: #247ee9;
```
Comment (Maxim-Mazurok) [inline website/style.css]: Can you change it to `#106ad5` to increase contrast and comply with [WCAG](https://www.w3.org/TR/WCAG20-TECHS/G18), please?
Text in the element `html > body > main > table > tbody > tr:nth-child(2) > th > a` has a contrast ratio of 3.52:1, while expected a contrast ratio of at least 4.5:1.

And also create a new rule for `pre a` with `color`: `#001631` for the same reason?
Text in the element `html > body > main > pre:nth-child(5) > a` has a contrast ratio of 1:1 (approximately, I suppose), while expected a contrast ratio of at least 4.5:1.

File: website/style.css [hunk 4]
```diff
@@ -1,74 +1,86 @@
...
+
+h2:hover a,
+h3:hover a  {
+  display: inline;
 }
 
 pre {
-  background: #ddd;
+  /* background: rgba(36, 126, 233, 0.03); */
+  background: rgba(36, 126, 233, 0.1);
```
Comment (Maxim-Mazurok) [inline website/style.css]: Can you change it to `#161616` to increase contrast and comply with [WCAG](https://www.w3.org/TR/WCAG20-TECHS/G18), please?
Text in the element `html > body > main > pre:nth-child(12)` has a contrast ratio of 2.43:1, while expected a contrast ratio of at least 4.5:1.

File: website/manual.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
+<!-- Copyright 2018-2019 the Deno authors. All rights reserved. MIT license. -->
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Deno Manual</title>
+    <link rel="stylesheet" href="style.css" />
+    <meta
+      content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=0"
```
Comment (Maxim-Mazurok) [inline website/manual.html]: Please, let's not block users from scaling the website, especially critical on mobile devices in terms of accessibility.
[Never use maximum-scale=1.0](https://a11yproject.com/posts/never-use-maximum-scale/)
[Interfering with a user agent's ability to zoom](https://www.w3.org/WAI/WCAG21/Understanding/reflow.html#failure)
So, let's remove `maximum-scale=1.0` and `user-scalable=0` from every page.

Comment (Maxim-Mazurok) [reply website/manual.html]: I have already opened https://github.com/denoland/deno/pull/1734 to fix that on `index.html`, but please, let's not add it to the `manual.html`.
`all_benchmark.html` already has only `width=device-width, initial-scale=1.0` as expected.

File: website/style.css (hunk 3 above)
Comment (ry) [reply website/style.css]: done

File: website/style.css (hunk 4 above)
Comment (ry) [reply website/style.css]: You mean for the `color:` right?

File: website/manual.html (hunk 5 above)
Comment (ry) [reply website/manual.html]: Done

File: website/style.css (hunk 4 above)
Comment (Maxim-Mazurok) [reply website/style.css]: Yes, sorry, that's right.

File: website/style.css [hunk 6]
```diff
@@ -1,74 +1,91 @@
-
 body {
-  color: #111;
+  color: #444;
   background: #f0f0f0;
-  margin: 1em;
-  font-family: Arial;
-  font-size: 20px;
+  padding: 32px 0;
```
Comment (hayd) [inline website/style.css]: IMO this should be 0;

Comment (ry) [reply website/style.css]: fixed.

**Pull request merged**